Source/WebCore/page/FrameView.cpp

```diff
@@ -2320,6 +2324,7 @@ void FrameView::maintainScrollPositionAtAnchor(ContainerNode* anchorNode)
         return;
 
     cancelScheduledScrollToFocusedElement();
```

Comment:
I wonder if we should factor all of these out into "cancel scheduled scrolls" function so we only have to add them in one place? But maybe this is OK.

Reply:
... but now that I scroll down, there are so many... so maybe do!